target/linux/ipq806x/base-files/etc/board.d/01_leds

```diff
@@ -21,6 +21,10 @@ edgecore,ecw5410)
 	ucidef_set_led_wlan "wlan2g" "WLAN2G" "green:wlan2g" "phy1tpt"
 	ucidef_set_led_wlan "wlan5g" "WLAN5G" "green:wlan5g" "phy0tpt"
 	;;
+extreme,ap3935)
+    ucidef_set_led_netdev "wan" "wan" "yellow:lan1" "eth0"
+    ucidef_set_led_netdev "lan" "lan" "yellow:lan2" "eth1"
+    ;;
```

Comment:
Minor thing: please use tabs to indent (to match the rest of the file)

Reply:
You are still using spaces

Reply:
Aw crap. Everytime I edit, it goes back to spaces unless I manually override.

Fixed again.